index.html

```diff
@@ -217,7 +217,7 @@ <h3 class="white-text center-align thin">New Top Level Domains!</h3>
 		<div class="col s12">
       <br /><br />
 			<h3 class="center-align thin">Donate</h3>
-			<p class="center-align">We provide all this completely free of charge! Please, if you are able, volunteer to <a href="http://wiki.opennicproject.org/Tier2ServerConfig" target="_blank">run a Tier 2 server</a> or feel welcome to make a donation:</p>
+			<p class="center-align">We provide all this completely free of charge! Please, if you are able, consider volunteering to run a Tier 2 server or feel welcome to make a donation:</p>
```

Comment:
Currently, removing the old wiki URL is the only solution to fix this dead link, but we should look into creating this page in the new wiki asap.